docs/editor/profiles.md

```diff
@@ -0,0 +1,169 @@
+---
+Order: 15
+Area: editor
+TOCTitle: Profiles
+ContentId: a65efc48-5a2d-4c7d-bd23-03f0393b53f6
+PageTitle: Profiles in Visual Studio Code
+DateApproved: 2/2/2023
+MetaDescription: Expand your development workflow with task integration in Visual Studio Code.
+---
+# Profiles in Visual Studio Code
+
+Visual Studio Code has 100s of [settings](/docs/getstarted/settings.md), 1000s of [extensions](/docs/editor/extension-marketplace.md), and unnumerable ways to adjust the UI layout to customize the editor. VS Code **Profiles** let you create sets of customizations and quickly switch between them or share them with others. This topic explains how to create, modify, export, and import profiles.
+
+## Create a Profile
+
+VS Code treats your current configuration as the **Default Profile**. As you modify settings, install extensions, or open and close views and panels, these customizations are tracked in the Default Profile.
+
+To create a new profile, you can use the **File** > **Preferences** > **Profiles** > **Create Profile** menu item (**Code** > > **Preferences** > **Profiles** > **Create Profile** on macOS).
+
+![Create Profile command](images/profiles/create-profile.png)
+
+You can create a new profile based on the current profile (**Profiles: Create from Current Profiles**) or create an Empty Profile. An Empty Profile includes no user customizations (settings, extensions, snippets, etc.) but with the UI state (view layouts and sizing) preserved.
+
+You can also access the Profile command menu via the **Manage** gear button in the lower right of the Activity bar.
+
+![Create Profile command via the Manage button in the Activity bar](images/profiles/create-profile-via-manage.png)
+
+The **Profiles: Create Profile** command is also available in the Command Palette (`kb(workbench.action.showCommands)`).
+
+Once you choose whether to create a new profile based on the current profile or an empty profile, you are prompted to enter a name for the new profile.
+
+### Check the current profile
+
+The current profile name is displayed in several places in the VS Code UI:
+
+* Title bar
+* **File** > **Preferences** > **Profiles**
+* **Manage** gear button hover
+
+If you are still using the Default Profile, no profile name is displayed.
+
+The **Manage** gear button displays a badge with the first two letters of the active profile so you can quickly check which profile you are running.
+
+![Manage gear displaying "DE' to indicate that the user's 'Demo' profile is active](images/profiles/profile-gear-two-letters.png)
+
+### Edit a profile
+
+You can edit a profile just as you would normally change any VS Code configuration. You can install/uninstall/disable extensions, change settings, and adjust snippets like normal. These changes are stored in your currently active profile.
```

Comment:
Also mention about UI layout customizations like moving views around, say moving view to secondary side bar, hiding views, view containers or actions.